workflow/scripts/wrapper.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+import pandas as pd
+import glob
+import shutil
+import os
+import yaml
+
+with open('config/wrapper_config.yaml', 'r') as f:
+    config = yaml.safe_load(f)
+
+# Get paths from config file
+inpath = config['input_dir']    # Input path containing barcode folders with fastq files
+outpath = config['output_dir']    # Output path for concatenated fastq files
+sample_sheet = pd.read_csv(config['sample_sheet'])    # Sample sheet following Nanopore naming convention
+
+# Create output directory if it does not exist (but is defined in the config file)
+if not os.path.exists(outpath):
+    os.makedirs(outpath)
+
+# Check if there is a negative control in the sample sheet
+if not sample_sheet['type'].str.contains('neg', case=False, na=False).any():
+    raise ValueError("No negative control could be found, please check input data. Aborting script.")
+
+
+def concatenate(sample_sheet, inpath, outpath):
```

Comment:
This would be good if you add some comments in the code. E.g. what the smaller steps do.